docs/compiling-css-with-gulp.md

```diff
@@ -0,0 +1,21 @@
+# Compiling CSS with Gulp
```

Comment:
This is good, we could also use docs about bear skin and bear coat in general. To answer the question, for example, which one should i be using?